target/scripts/update-check.sh

```diff
@@ -17,7 +17,8 @@ fi
 
 while true; do
   # get remote version information
-  LATEST=$(curl -Lsf "${VERSION_URL}")
+  # JSON response provides a field for the release tag, the `v` prefix is removed with `[1:]`
+  LATEST=$(curl -sfL -H 'accept: application/json' "${VERSION_URL}" | jaq -r '.tag_name[1:]')
```

Comment:
We might want to alias `jq` to `jaq`? This way users could transparently use `jq` too. But I would not insist on this.

Reply:
> This way users could transparently use `jq` too.

Same policy as usual, wait until there is sufficient interest expressed from users. If there's only a few they are probably quite happy to use `jaq`.

This is for internal use, we don't document it as available officially, so we're also free to change it to something else like `yq` if `jaq` isn't as good of a fit elsewhere I'd like to use it.